Description: Set oxygen as default fallback
 until the FIXME is done. That's all i want to do for now.
 oxygen is merged upstream so the patch become obsolete with the
 upcoming release.
Author: Alf Gaida <agaida@siduction.org>
Bug-Debian: https://bugs.debian.org/826311
Last-Update: 2016-06-04

--- pcmanfm-qt-0.11.0.orig/pcmanfm/settings.cpp
+++ pcmanfm-qt-0.11.0/pcmanfm/settings.cpp
@@ -163,7 +163,7 @@ bool Settings::loadFile(QString filePath
   if(fallbackIconThemeName_.isEmpty()) {
     // FIXME: we should choose one from installed icon themes or get
     // the value from XSETTINGS instead of hard code a fallback value.
-    fallbackIconThemeName_ = "elementary"; // fallback icon theme name
+    fallbackIconThemeName_ = "oxygen"; // fallback icon theme name
   }
   suCommand_ = settings.value("SuCommand", "lxqt-sudo %s").toString();
   setTerminal(settings.value("Terminal", "xterm").toString());
